src/translate_c.zig

```diff
@@ -1866,7 +1866,13 @@ fn transDeclRefExpr(
     const value_decl = expr.getDecl();
     const name = try c.str(@ptrCast(*const clang.NamedDecl, value_decl).getName_bytes_begin());
     const mangled_name = scope.getAlias(name);
-    var ref_expr = try Tag.identifier.create(c.arena, mangled_name);
+    var ref_expr = if (mem.eql(u8, mangled_name, "_"))
+        try Tag.field.create(c.arena, .{
+            .lhs = Tag.this.init(),
+            .rhs = try Tag.string_literal.create(c.arena, "\"_\""),
```

Comment:
Does this string literal (`"\"_\""`) need to be allocated in the arena? (e.g. `mem.dupe(c.arena, u8, "\"_\"")`)? Other ones are but I assume that is because they come from clang and have a different lifetime.

Reply:
No it just needs to exist until the intermediate ast is rendered and static lifetime will suffice for that.